README.md

```diff
@@ -140,17 +140,19 @@ from GitHub. You may make your own fork project from our project.
 ### 1. Debian/Ubuntu
 
 - gcc
-- libncurses-dev
-- libreadline-dev
 - make
-- openssl-dev
+- libreadline-dev

```

Comment:
Why did you move those three lines to the bottom of the list? It was by intention that the list is alphabetically sorted

Reply:
ok.I will correct.this sort is the package I install for a newly insalled system, if compile and link error, I will install by the feedback message. all by my test.